ee-9/build/pom.xml

```diff
@@ -88,7 +88,6 @@
                             <plugin-options>
                                 <jboss-maven-dist/>
                                 <jboss-jakarta-transform-artifacts>false</jboss-jakarta-transform-artifacts>
```

Comment:
Yo ucan remove all these occurences, you disabled transformation when building the feature-pack, so jakarta-transformation is disabled by default.